strawberryfields/engine.py

```diff
@@ -168,11 +174,108 @@ def _run_program_locally(self, prog, **kwargs):
         Returns:
             list[Command]: commands that were applied to the backend
         """
+
+    def _run(self, program, *, compile=True, **kwargs):
+        """Execute the given programs by sending them to the backend.
```

Comment:
We should clarify what the scope of `program` is when it is a sequence. Is it a list of programs, each of which should be run independently one after the other, or is it a list of program segments which should be linked together sequentially (without returning intermediate measurements) as part of a longer computation?

Reply:
It's the latter, I'll clarify it.